DangerRulesTest/ViewController.swift

```diff
@@ -10,6 +10,8 @@ import UIKit
 class ViewController: UIViewController {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/mayur-gauswami/DangerIntegration/blob/b9e4b7cd0ef640f20b523bf5a8fb40c606369e56/DangerRulesTest/ViewController.swift#L10"></span>Consider using final for this class or use a struct (final_class)</td>
    </tr>
  </tbody>
</table>